### PR TITLE
Add option for Decoder to output f32.

### PR DESCRIPTION
**Pull request by WilliamTambellini**
Add option to add some metadata.
No change of default behavior.

### CODE REVIEW

File: Export_ONNX/F5_TTS/Export_F5.py [hunk 1]
```diff
@@ -256,9 +262,16 @@ def forward(self,
...
         # generated_signal = generated_signal * self.target_rms / torch.sqrt(torch.mean(generated_signal * generated_signal))  # Optional process
-        return (generated_signal.clamp(min=-1.0, max=1.0) * 32767.0).to(torch.int16)
-
+        #logging.info(f"sys maxint={sys.maxint}")
+        #logging.info(f"shrt_max={limits.shrt_max}")
+        if self.output_type == torch.int16:
+            max_int16 = float((1 << 15) - 1)  # Result: 32767.0
+            return (generated_signal.clamp(min=-1.0, max=1.0) * 32767.0).to(torch.int16)
+        if self.output_type == torch.float32:
+            return generated_signal
```
Comment (WilliamTambellini) [inline Export_ONNX/F5_TTS/Export_F5.py:273]: perhaps should we normalize here...

**Review by WilliamTambellini — COMMENTED** [review]
nit
